driver/src/driver/stablex_driver.rs

```diff
@@ -155,7 +155,8 @@ impl<'a> StableXDriver for StableXDriverImpl<'a> {
             Some(time_limit) => time_limit,
             None => {
                 self.metrics.auction_skipped(batch_to_solve);
-                return DriverResult::Skip(anyhow!("account retrieval exceeded time limit"));
```

Comment:
Maybe `Skip` here can also be misleading. Maybe the DriverResult should be ErrorRetryable, ErrorNonRetryable instead to make clear that Skip here is not benign

Reply:
I'll do this in a separate PR.